server/instances_test.go

```diff
@@ -110,7 +110,7 @@ func TestInstallInstance(t *testing.T) {
 
 			testInstance0 := &testInstance{
 				InstanceCommon: InstanceCommon{
-					InstanceID: mockInstance1URL,
+					InstanceID: mockInstance3URL,
```

Comment:
why this change?

Reply:
Testcases are failing without this change. I think there are some changes in the function